upload/admin/controller/report/statistics.php

```diff
@@ -218,4 +218,26 @@ public function review(): void {
 		$this->response->addHeader('Content-Type: application/json');
 		$this->response->setOutput(json_encode($json));
 	}
-}
\ No newline at end of file
+
+	public function comment(): void {
+		$this->load->language('report/statistics');
+
+		$json = [];
+
+		if (!$this->user->hasPermission('modify', 'report/statistics')) {
+			$json['error'] = $this->language->get('error_permission');
+		}
+
+		if (!$json) {
+			$this->load->model('report/statistics');
+			$this->load->model('blog/comment');
+
+			$this->model_report_statistics->editValue('comment', $this->model_blog_comment->getTotalCommentsAwaitingApproval());
```

Comment:
This line would only pull the comments that are not approved yet vs. the ones that would be already approved.

Reply:
That's it with the product review!

Reply:
Yes, correct.